cv/qc-cv.obo

```diff
@@ -18,1967 +18,341 @@ import: https://raw.githubusercontent.com/ISA-tools/stato/dev/releases/1.4/stato
-comment: Estimates very early peak broadening.
-is_a: MS:4000006 ! table
-is_a: MS:4000009 ! ID based
-is_a: MS:4000022 ! chromatogram metric
-relationship: has_relation MS:4000013 ! single run based
-relationship: has_column: MS:4000108 ! Retention time
-relationship: has_column: MS:4000239 ! Fraction
+name: number of identified peptides
+def: "Number of identified peptides at given FDR threshold." [PSI:QC]
+is_a: MS:4000003 ! single value
+relationship: has_metric_category MS:4000008 ! ID based metric
+relationship: has_relation QC:4000012 ! single run based metric
+relationship: has_units UO:0000189 ! count unit
 
 [Term]
 id: MS:4000072
-name: Interquartile RT period for peptide identifications
-def: "The interquartile retention time period, in seconds, for all peptide identifications over the complete run." [PSI:QC]
-comment: Longer times indicate better chromatographic separation.
+name: number of identified distinct peptides
+def: "Number of identified peptides without duplicate count (or modifications) at given FDR threshold." [PSI:QC]
 is_a: MS:4000003 ! single value
-is_a: MS:4000009 ! ID based
-is_a: MS:4000022 ! chromatogram metric
-relationship: has_relation MS:4000013 ! single run based
+relationship: has_metric_category MS:4000008 ! ID based metric
+relationship: has_relation QC:4000012 ! single run based metric
+relationship: has_units UO:0000189 ! count unit
 
 [Term]
 id: MS:4000073
-name: Peptide identification rate of the interquartile RT period
-def: "The identification rate of peptides for the interquartile retention time period, in peptides per second." [PSI:QC]
-comment: Higher rates indicate efficient sampling and identification.
+name: number of proteins and groups from all PSM
+def: "Number of proteins identified from all PSM at given FDR threshold, groups count as one." [PSI:QC]
 is_a: MS:4000003 ! single value
-is_a: MS:4000009 ! ID based
-is_a: MS:4000022 ! chromatogram metric
-relationship: has_relation MS:4000013 ! single run based
-relationship: has_relation MS:4000072 ! Interquartile RT period for peptide identifications
+relationship: has_metric_category MS:4000008 ! ID based metric
+relationship: has_relation QC:4000012 ! single run based metric
+relationship: has_units UO:0000189 ! count unit
 
 [Term]
 id: MS:4000074
-name: Median MS1 peak FWHM for peptides
-def: "Median of all MS1 peak widths at half maximum (FWHM) for all identified peptides, in seconds" [PSI:QC]
-comment: Sharper peak widths (i.e. smaller values) indicate better chromatographic resolution.
+name: number of distinct proteins from all PSM
+def: "Number of distinct proteins without duplicate count or groups from all PSM at given FDR threshold." [PSI:QC]
```

Comment:
... from teh comment before: and here  count all accessions in the respective protein groups?